src/routes/load/components/DetailsForm/index.tsx

```diff
@@ -95,7 +96,7 @@ const DetailsForm = ({ errors, form }: DetailsFormProps): React.ReactElement =>
             placeholder="Name of the Safe*"
             text="Safe name"
             type="text"
-            validate={composeValidators(required, minMaxLength(1, 50))}
+            validate={composeValidators(required, minMaxLength(1, 50), validAddressBookName)}
```

Comment:
We could make `minMaxLength(1, 50)` part of `validAddressBookName`.

Reply:
done